### PR TITLE
fix!: remove nuxt as default test environment

### PR DESCRIPTION
**Pull request by danielroe**
BREAKING CHANGE: you now must set a default `test.environment` in your `vitest.config`, or alternatively opt-in per test file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +45,39 @@ export default defineConfigWithNuxt({
...
+```js
+// @vitest-environment nuxt
+import { test } from 'nuxt-vitest'
+
+test('my test', () => {
+  // ... test with Nuxt environment!
+})
+```
+
+Finallty, you can set `envoironment: 'nuxt'`, to enable Nuxt environment for **all tests**.
```
Comment (dsvgl) [inline README.md:65]:
```suggestion
Finally, you can set `envoironment: 'nuxt'`, to enable Nuxt environment for **all tests**.
```

**Review by dsvgl — CHANGES_REQUESTED** [review]

File: playground/vitest.config.ts [hunk 2]
```diff
@@ -2,10 +2,9 @@ import { defineConfigWithNuxt } from 'nuxt-vitest/config'
 
 export default defineConfigWithNuxt({
```
Comment (danielroe) [inline playground/vitest.config.ts:3]: Should we rename this method? I wonder if it's a bit confusing...

Comment (antfu) [reply playground/vitest.config.ts:3]: It **is** with Nuxt (the config, the pipeline, etc.), just not the runtime environment by default. Do you have any better name in mind?

Comment (danielroe) [reply playground/vitest.config.ts:3]: That's true. And no, I don't 😆

**Pull request merged**